src/ucp/wireup/wireup_cm.c

```diff
@@ -1309,26 +1333,41 @@ ucs_status_t ucp_ep_cm_connect_server_lane(ucp_ep_h ep,
                                UCT_EP_PARAM_FIELD_CONN_REQUEST              |
                                UCT_EP_PARAM_FIELD_USER_DATA                 |
                                UCT_EP_PARAM_FIELD_SOCKADDR_CB_FLAGS         |
-                               UCT_EP_PARAM_FIELD_SOCKADDR_PACK_CB          |
                                UCT_EP_PARAM_FIELD_SOCKADDR_NOTIFY_CB_SERVER |
-                               UCT_EP_PARAM_FIELD_SOCKADDR_DISCONNECT_CB;
+                               UCT_EP_PARAM_FIELD_SOCKADDR_DISCONNECT_CB    |
+                               UCT_EP_PARAM_FIELD_PRIV_DATA                 |
+                               UCT_EP_PARAM_FIELD_PRIV_DATA_LENGTH;
 
     uct_ep_params.cm                 = worker->cms[cm_idx].cm;
     uct_ep_params.user_data          = ep;
     uct_ep_params.conn_request       = uct_conn_req;
     uct_ep_params.sockaddr_cb_flags  = UCT_CB_FLAG_ASYNC;
-    uct_ep_params.sockaddr_pack_cb   = ucp_cm_server_priv_pack_cb;
     uct_ep_params.sockaddr_cb_server = ucp_cm_server_conn_notify_cb;
     uct_ep_params.disconnect_cb      = ucp_cm_disconnect_cb;
+    status = ucp_ep_server_init_priv_data(ep, dev_name,
+                                          &uct_ep_params.private_data,
+                                          &uct_ep_params.private_data_length);
+    if (status != UCS_OK) {
+        goto out;
+    }
 
     status = uct_ep_create(&uct_ep_params, &uct_ep);
+    ucs_free((void *)uct_ep_params.private_data);
```

Comment:
minor: excessive whitespace between type and asterisk, it should be:
```suggestion
    ucs_free((void*)uct_ep_params.private_data);
````